src/PointwiseFunctions/InitialDataUtilities/InitialData.hpp

```diff
@@ -0,0 +1,28 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <pup.h>
+#include <pup_stl.h>
+
+#include "Parallel/CharmPupable.hpp"
+
+namespace InitialDataUtilities {
+/*!
+ * \ingroup InitialDataGroup
+ * \brief The abstract base class for initial data of evolution systems.
+ */
+class InitialData : public PUP::able {
```

Comment:
I think let's put this in the `evolution` namespace. It's evolution-specific and it's a lot more intuitive to write `evolution::InitialData` and `evolution::Tags::InitialData`.

Reply:
It seems that throughout the code in general the namespace follows the name of the library, so I think it'd be better keep `InitialDataUtilites` since we add a new library here -- if we need to mark something as evolution-specific, how about putting these into `InitialDataUtilities::evolution`?